rapids/rapids.sh

```diff
@@ -59,6 +54,34 @@ function execute_with_retries() {
   return 1
 }
 
+function install_dask_rapids() {
+  local -r base=$(conda info --base)
+  local -r pinned=${base}/conda-meta/pinned
+  local -r mamba_env=mamba
+  
+  # Using mamba significantly reduces the conda solve-time. Create a separate conda
+  # environment with mamba installed to manage installations.
+  conda create -y -n ${mamba_env} -c conda-forge mamba
+
+  # RAPIDS releases require fixed PyArrow versions. Unpin PyArrow to solve
+  # for new environment.
+  sed -i 's/pyarrow [\.0-9\*]*/pyarrow/g' ${pinned}
+
+  # Install RAPIDS and cudatoolkit. Use mamba in new env to resolve base environment
+  installer=${base}/envs/${mamba_env}/bin/mamba
+  $installer install -y \
+    -c "rapidsai" -c "nvidia" -c "conda-forge" -c "defaults" \
+    "cudatoolkit=${CUDA_VERSION}" "rapids=${RAPIDS_VERSION}" \
+    -p ${base}
+
+  # Repin PyArrow with new version
+  local -r version=$(conda list pyarrow | grep -E pyarrow 2>&1 | sed -n 's/pyarrow[[:blank:]]\+\([0-9\.]\+\).*/\1/p')
+  sed -i 's/pyarrow/pyarrow '"${version}"'\.\*/g' ${pinned}
```

Comment:
What PyArrow version do we need?

Reply:
Each version of RAPIDS requires different yet exact versions of PyArrow, built with cuda support:
RAPIDS 0.14.x requires PyArrow 0.15.0
0.15.x (current) requires 0.17.1
0.16.x (in preview) requires 1.0.1